wallet/src/ui/app/ApiProvider.ts

```diff
@@ -11,20 +11,43 @@ export enum API_ENV {
     staging = 'staging',
 }
 
+// fallback default to devNet if not set
+const defaultRPCENV = {
```

Comment:
I don't think we need this. Without this we have all the endpoints to one place `.env.default` (and maybe `.env` if we want to override something). We just need to make sure we have all the env variables defined. We can add some tests if you want but I think relaying on [`getDefaultAPI`](https://github.com/MystenLabs/sui/pull/2697/files#diff-68ee8f4925ae70fbfe3afc00655b3d2c1ce7b2bc7d968b57499a1ba70ae69551R64) to throw, should be enough. Also with this we don't have a way to detect that we have't defined an endpoint for some envs and using for different environments devNet as default doesn't sound right